serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -1 +1,89 @@
+
+This section explains how to configure Smallrye channels for a Kogito Serverless Workflow using event definitions. This info is extracted from link:{kogito_messaging_url}[Kogito documentation]
+
+Channels are configured by using properties. The pattern for these properties is `mp.messaging.[incoming|outgoing].<channel name>.<property_name>`.
+
+Kogito serverless workflow allows different channel mapping strategies:
+
+* Define one default incoming channel to receive all the incoming messages and one default outgoing channel to store all the published messages.
+
+* Define a channel for each Cloud Event Type so that every message type has a dedicated channel.
+
+* Define a channel for certain Cloud Event Types and let the not mapped Cloud Event types to use the default incoming or outgoing channel.
+
+Kogito first searches for channel name equals to Cloud Event type in the properties. If found, it uses that channel for that Cloud Event type. If not, it searches for default channel definition. If also not existing, then an error will be reported.
+
+The name for the default incoming channel is `kogito_incoming_stream` and for the default outgoing channel is `kogito_outgoing_stream`.
+
+You can change the default incoming and outgoing topic name as follows:
+
+[source,properties]
+----
+kogito.addon.messaging.incoming.defaultName=<default channel name>
+kogito.addon.messaging.outgoing.defaultName=<default channel name>
+----
+
+This is the full list of link:{kafka_connector_properties_url}[properties] you can use for a channel when using Kafka connector.
+
+There are two mandatory properties that you need to setup for every channel defined:
+
+* `connector`, which has to be set to `smallrye-kafka`
+* Depending on the channel being input or output: 
+** if incoming, `value.deserializer`. Unless you have specific marshaling needs, will be set to `org.apache.kafka.common.serialization.ByteArrayDeserializer` or `org.apache.kafka.common.serialization.StringDeserializer`
+** if outgoing, `value.serializer`. Unless you have specific marshaling needs, will be set to `org.apache.kafka.common.serialization.ByteArraySerializer` or `org.apache.kafka.common.serialization.StringSerializer`
+
+Another relevant, although optional, property is `topic`. If present, it should contain the Kafka topic name to be used for that channel. If missing, the channel name will be used as topic name. 
+
+=== Channel Mapping Example 
+
+Let illustrate the previous explanation by reviewing the properties configured in the link:{configuration_example_url}[Callback example].
+This example uses two Cloud Event types: `wait`, which is incoming, and `resume`, which is outgoing. 
+
+Kafka topic names matches the cloud event types, therefore it makes sense to chose one channel per Cloud Event type mapping strategy. This implies two channels must be configured. Since the channel name matches the cloud event type, we do not need to indicate the optional `topic` property.
```

Comment:
```suggestion
Kafka topic names matches the cloud event types, therefore it makes sense to chose one channel per Cloud Event type mapping strategy. This implies two channels to be configured. Since the channel name matches the cloud event type, we do not need to indicate the optional `topic` property.
```

Reply:
Hmmm, I prefer to use must to emphasize this is mandatory step